README.md

```diff
@@ -8,8 +8,20 @@
 
 ### Demo
 
+Installation & Help message
+[![asciicast](https://asciinema.org/a/616538.svg)](https://asciinema.org/a/616538)
+
+Plain JSON files diff:
 [![asciicast](https://asciinema.org/a/602583.svg)](https://asciinema.org/a/602583)
+
+Plain YAML files diff:
 [![asciicast](https://asciinema.org/a/604893.svg)](https://asciinema.org/a/604893)
+
+Nested JSON files diff:
 [![asciicast](https://asciinema.org/a/612701.svg)](https://asciinema.org/a/612701)
+
+Plain diff format:
 [![asciicast](https://asciinema.org/a/612992.svg)](https://asciinema.org/a/612992)
```

Comment:
this asciinema has too many runs :) ideally, your asciinema should have only one execution (no need to do `gendiff` and `gendiff -h` before doing actual execution)